projects/dev/src/app/shared/services/api/group.service.ts

```diff
@@ -92,22 +91,22 @@ export class GroupService {
       .pipe(catchError(this.handleError));
   }
 
-  acceptJoinRequest(id, group_ids) {
+  acceptJoinRequest(id, group_ids): Observable<RequestActionResponse> {
```

Comment:
Types for input parameters!
For acceptJoinRequest, rejectJoinRequest and getManagedRequests

Reply:
fixed.